front/src/components/Table/components/SettingsIcon/styles.tsx

```diff
@@ -0,0 +1,22 @@
+import { makeStyles } from "@material-ui/core/styles";
+import styled from "@emotion/styled";
+import { MenuItem as MaterialMenuItem } from "@material-ui/core";
+
+import { theme } from "assets/theme";
+
+const { media, css } = theme;
+
+export const MenuItem = styled(MaterialMenuItem)`
+  ${media.sm} {
+    ${css.md_text}
```

Comment:
takie coś jest średnie, bo nic to nie mowi na pierwszy rzut oka. Lepiej zostawić parametr, tj. font-size i brac skads tylko sama wartosc, typu
font-size: ${font.size.md}

Reply:
Racja.
Done.